DQMOffline/Configuration/python/DQMOfflineHeavyIons_cff.py

```diff
@@ -30,12 +30,12 @@
                                        DQMMessageLogger )
 
 from DQMOffline.Muon.muonMonitors_cff import *
-diMuonHistos.etaBin = cms.int32(70) #dimuonhistograms mass, bin
-diMuonHistos.etaBBin = cms.int32(70)
-diMuonHistos.etaEBin = cms.int32(70)
-diMuonHistos.etaBinLM = cms.int32(12)
-diMuonHistos.etaBBinLM = cms.int32(12)
-diMuonHistos.etaEBinLM = cms.int32(12)
+diMuonHistos.etaBin = cms.int32(175) #dimuonhistograms mass, bin
```

Comment:
Hi @davidlange6 @slava77 , 
Thanks for the comment. Those 3 lines can be cleared from this file and moved to CFI itself. But it would be better to let this file as it is now and determine later when next heavy ion run configuration discussion happens. We would like to wrap this PR rather soon for upcoming pPb runs, and we will make another PR after run ends.